### ISSUE TITLE
AB not starting after running "activity-browser"

### ISSUE THREAD

**Issue by AlJuel** [LCA-ActivityBrowser/activity-browser#979]
Hey there,

activity-browser is not starting after running the "actvitity-browser" command.

However, there is also no error message - simply nothing happens!

**Here is my conda list:**
```
# packages in environment at C:\Users\alexander\miniconda3\envs\ab:
#
# Name                    Version                   Build  Channel
activity-browser          2.7.6              pyhd8ed1ab_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
asteval                   0.9.29             pyhd8ed1ab_0    conda-forge
astunparse                1.6.3              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
blinker                   1.6.2              pyhd8ed1ab_0    conda-forge
brightway2                2.4.3              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                hcfcfb64_8    conda-forge
brotli-bin                1.0.9                hcfcfb64_8    conda-forge
brotli-python             1.0.9            py39h99910a6_8    conda-forge
brotlicffi                1.0.9.2          py39h99910a6_3    conda-forge
bw2analyzer               0.10               pyhd8ed1ab_0    conda-forge
bw2calc                   1.8.2            py39hcbf5309_0    conda-forge
bw2data                   3.6.6              pyhd8ed1ab_0    conda-forge
bw2io                     0.8.8              pyhd8ed1ab_1    conda-forge
bw2parameters             1.1.0              pyhd8ed1ab_0    conda-forge
bw_migrations             0.2                pyhd8ed1ab_0    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2023.5.7             h56e8100_0    conda-forge
certifi                   2023.5.7           pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h68f70e3_3    conda-forge
charset-normalizer        3.1.0              pyhd8ed1ab_0    conda-forge
click                     8.1.3           win_pyhd8ed1ab_2    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
contourpy                 1.0.7            py39h1f6ef14_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dill                      0.3.6              pyhd8ed1ab_1    conda-forge
docopt                    0.6.2                      py_1    conda-forge
eidl                      1.5.1              pyhd8ed1ab_1    conda-forge
eight                     1.0.1              pyhd8ed1ab_4    conda-forge
et_xmlfile                1.1.0              pyhd8ed1ab_0    conda-forge
fasteners                 0.17.3             pyhd8ed1ab_0    conda-forge
flask                     2.3.2              pyhd8ed1ab_0    conda-forge
fonttools                 4.39.4           py39ha55989b_0    conda-forge
freetype                  2.12.1               h546665d_1    conda-forge
future                    0.18.3             pyhd8ed1ab_0    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
glib                      2.76.3               h12be248_0    conda-forge
glib-tools                2.76.3               h12be248_0    conda-forge
gst-plugins-base          1.22.3               h001b923_1    conda-forge
gstreamer                 1.22.3               h6b5321d_1    conda-forge
icu                       72.1                 h63175ca_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.6.0              pyha770c72_0    conda-forge
importlib-resources       5.12.0             pyhd8ed1ab_0    conda-forge
importlib_metadata        6.6.0                hd8ed1ab_0    conda-forge
importlib_resources       5.12.0             pyhd8ed1ab_0    conda-forge
inflate64                 0.3.1            py39ha55989b_3    conda-forge
intel-openmp              2023.1.0         h57928b3_46319    conda-forge
itsdangerous              2.1.2              pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
kiwisolver                1.4.4            py39h1f6ef14_1    conda-forge
krb5                      1.20.1               heb0366b_0    conda-forge
lcms2                     2.15                 h3e3b177_1    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libblas                   3.9.0              17_win64_mkl    conda-forge
libbrotlicommon           1.0.9                hcfcfb64_8    conda-forge
libbrotlidec              1.0.9                hcfcfb64_8    conda-forge
libbrotlienc              1.0.9                hcfcfb64_8    conda-forge
libcblas                  3.9.0              17_win64_mkl    conda-forge
libclang                  16.0.5          default_heb8d277_0    conda-forge
libclang13                16.0.5          default_hc80b9e7_0    conda-forge
libdeflate                1.18                 hcfcfb64_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libglib                   2.76.3               he8f3873_0    conda-forge
libhwloc                  2.9.1            cpu_hadd60ae_5    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libjpeg-turbo             2.1.5.1              hcfcfb64_0    conda-forge
liblapack                 3.9.0              17_win64_mkl    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libsqlite                 3.42.0               hcfcfb64_0    conda-forge
libtiff                   4.5.0                h6c8260b_6    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp                   1.3.0                hcfcfb64_0    conda-forge
libwebp-base              1.3.0                hcfcfb64_0    conda-forge
libxcb                    1.15                 hcd874cb_0    conda-forge
libxml2                   2.11.4               hc3477c8_0    conda-forge
libxslt                   1.1.37               h6070c61_1    conda-forge
libzlib                   1.2.13               hcfcfb64_4    conda-forge
lxml                      4.9.1            py39hb82d6ee_0    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.1.3            py39ha55989b_0    conda-forge
matplotlib-base           3.7.1            py39haf65ace_0    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
mrio_common_metadata      0.2.1              pyhd8ed1ab_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multiprocess              0.70.14          py39ha55989b_3    conda-forge
multivolumefile           0.2.3              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
numpy                     1.23.5           py39hbccbffa_0    conda-forge
openjpeg                  2.5.0                ha2aaf27_2    conda-forge
openpyxl                  3.1.2            py39ha55989b_0    conda-forge
openssl                   3.1.1                hcfcfb64_1    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.2            py39h1679cfb_0    conda-forge
pathos                    0.3.0              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
peewee                    3.16.2           py39habf9222_0    conda-forge
pillow                    9.5.0            py39ha9166d5_1    conda-forge
pint                      0.21               pyhd8ed1ab_0    conda-forge
pip                       23.1.2             pyhd8ed1ab_0    conda-forge
platformdirs              3.5.1              pyhd8ed1ab_0    conda-forge
pooch                     1.7.0              pyha770c72_3    conda-forge
pox                       0.3.2              pyhd8ed1ab_0    conda-forge
ppft                      1.7.6.6            pyhd8ed1ab_0    conda-forge
psutil                    5.9.5            py39ha55989b_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
py7zr                     0.20.5             pyhd8ed1ab_0    conda-forge
pybcj                     1.0.1            py39ha55989b_2    conda-forge
pybcpy                    0.0.17             pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycryptodomex             3.18.0           py39ha55989b_0    conda-forge
pypardiso                 0.4.2              pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyperclip                 1.8.2              pyhd8ed1ab_2    conda-forge
pyppmd                    1.0.0            py39h99910a6_1    conda-forge
pyprind                   2.11.2          py39hcbf5309_1004    conda-forge
pyside2                   5.15.8           py39hdfd5cee_3    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.9.16          h4de0772_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      3_cp39    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pyxlsb                    1.0.10             pyhd8ed1ab_0    conda-forge
pyzstd                    0.15.7           py39h95af829_0    conda-forge
qt-main                   5.15.8              h2c8576c_12    conda-forge
qt-webengine              5.15.8               h5b1ea0b_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
salib                     1.4.7              pyhd8ed1ab_0    conda-forge
scipy                     1.10.1           py39hde5eda1_3    conda-forge
seaborn                   0.12.2               hd8ed1ab_0    conda-forge
seaborn-base              0.12.2             pyhd8ed1ab_0    conda-forge
setuptools                67.7.2             pyhd8ed1ab_0    conda-forge
setuptools-scm            7.1.0              pyhd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
stats_arrays              0.6.5              pyhd8ed1ab_0    conda-forge
statsmodels               0.14.0           py39hbaa61f9_1    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tbb                       2021.9.0             h91493d7_0    conda-forge
texttable                 1.6.7              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.6.3                hd8ed1ab_0    conda-forge
typing_extensions         4.6.3              pyha770c72_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
unicodedata2              15.0.0           py39ha55989b_0    conda-forge
unidecode                 1.3.6              pyhd8ed1ab_0    conda-forge
urllib3                   2.0.2              pyhd8ed1ab_0    conda-forge
vc                        14.3                hb25d44b_16    conda-forge
vc14_runtime              14.34.31931         h5081d32_16    conda-forge
voluptuous                0.13.1             pyhd8ed1ab_0    conda-forge
vs2015_runtime            14.34.31931         hed1258a_16    conda-forge
werkzeug                  2.3.4              pyhd8ed1ab_0    conda-forge
wheel                     0.40.0             pyhd8ed1ab_0    conda-forge
whoosh                    2.7.4            py39hcbf5309_7    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
wrapt                     1.15.0           py39ha55989b_0    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xlsxwriter                3.1.2              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.11               hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zipfile-deflate64         0.2.0            py39ha55989b_3    conda-forge
zipp                      3.15.0             pyhd8ed1ab_0    conda-forge
zstd                      1.5.2                h12be248_6    conda-forge
```

**Comment by Zoophobus** [comment]
Hi @AlJuel,

this is very weird, I've just tried a new install and it works fine so I'm curious what's happened. Can you tell me if you recently changed the environment, ie upgraded the AB, or if this is a new install?

**Comment by AlJuel** [comment]
Hi @Zoophobus,

it's a new install.  I have also tried to reinstall miniconda, and to set up a new environment - but that did not help.

**Comment by marc-vdm** [comment]
@AlJuel Could you please re-install the environment again -or if you have the terminal still open skip that- and copy-paste the entire output from `conda create -n ab activity-browser` until it finishes? That may help us understand if something is going wrong during installation.

Also, could you estimate roughly how long the full process takes for your computer? No worries about exact seconds, but roughly how many minutes it takes.

**Comment by antoine7878** [comment]
Hi,

I have what seems to be the same issue as @AlJuel. I first noticed it last friday: new installations of AB do not launch. Installations made couple of weeks ago are still working fine. 
I just did a fresh install, it took about 20 min with the following output: 


```
(base) PS C:\Users\antoi> conda create -n ab2 activity-browser
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\antoi\miniconda3\envs\ab2

  added / updated specs:
    - activity-browser


The following NEW packages will be INSTALLED:

  activity-browser   conda-forge/noarch::activity-browser-2.7.6-pyhd8ed1ab_0
  appdirs            conda-forge/noarch::appdirs-1.4.4-pyh9f0ad1d_0
  arrow              conda-forge/noarch::arrow-1.2.3-pyhd8ed1ab_0
  asteval            conda-forge/noarch::asteval-0.9.29-pyhd8ed1ab_0
  astunparse         conda-forge/noarch::astunparse-1.6.3-pyhd8ed1ab_0
  beautifulsoup4     conda-forge/noarch::beautifulsoup4-4.12.2-pyha770c72_0
  blinker            conda-forge/noarch::blinker-1.6.2-pyhd8ed1ab_0
  brightway2         conda-forge/noarch::brightway2-2.4.3-pyhd8ed1ab_0
  brotli             conda-forge/win-64::brotli-1.0.9-hcfcfb64_8
  brotli-bin         conda-forge/win-64::brotli-bin-1.0.9-hcfcfb64_8
  brotli-python      conda-forge/win-64::brotli-python-1.0.9-py39h99910a6_8
  brotlicffi         conda-forge/win-64::brotlicffi-1.0.9.2-py39h99910a6_3
  bw2analyzer        conda-forge/noarch::bw2analyzer-0.10-pyhd8ed1ab_0
  bw2calc            conda-forge/win-64::bw2calc-1.8.2-py39hcbf5309_0
  bw2data            conda-forge/noarch::bw2data-3.6.6-pyhd8ed1ab_0
  bw2io              conda-forge/noarch::bw2io-0.8.8-pyhd8ed1ab_1
  bw2parameters      conda-forge/noarch::bw2parameters-1.1.0-pyhd8ed1ab_0
  bw_migrations      conda-forge/noarch::bw_migrations-0.2-pyhd8ed1ab_0
  bzip2              conda-forge/win-64::bzip2-1.0.8-h8ffe710_4
  ca-certificates    conda-forge/win-64::ca-certificates-2023.5.7-h56e8100_0
  certifi            conda-forge/noarch::certifi-2023.5.7-pyhd8ed1ab_0
  cffi               conda-forge/win-64::cffi-1.15.1-py39h68f70e3_3
  charset-normalizer conda-forge/noarch::charset-normalizer-3.1.0-pyhd8ed1ab_0
  click              conda-forge/noarch::click-8.1.3-win_pyhd8ed1ab_2
  colorama           conda-forge/noarch::colorama-0.4.6-pyhd8ed1ab_0
  contourpy          conda-forge/win-64::contourpy-1.0.7-py39h1f6ef14_0
  cycler             conda-forge/noarch::cycler-0.11.0-pyhd8ed1ab_0
  dill               conda-forge/noarch::dill-0.3.6-pyhd8ed1ab_1
  docopt             conda-forge/noarch::docopt-0.6.2-py_1
  eidl               conda-forge/noarch::eidl-1.5.1-pyhd8ed1ab_1
  eight              conda-forge/noarch::eight-1.0.1-pyhd8ed1ab_4
  et_xmlfile         conda-forge/noarch::et_xmlfile-1.1.0-pyhd8ed1ab_0
  fasteners          conda-forge/noarch::fasteners-0.17.3-pyhd8ed1ab_0
  flask              conda-forge/noarch::flask-2.3.2-pyhd8ed1ab_0
  fonttools          conda-forge/win-64::fonttools-4.39.4-py39ha55989b_0
  freetype           conda-forge/win-64::freetype-2.12.1-h546665d_1
  future             conda-forge/noarch::future-0.18.3-pyhd8ed1ab_0
  gettext            conda-forge/win-64::gettext-0.21.1-h5728263_0
  glib               conda-forge/win-64::glib-2.76.3-h12be248_0
  glib-tools         conda-forge/win-64::glib-tools-2.76.3-h12be248_0
  gst-plugins-base   conda-forge/win-64::gst-plugins-base-1.22.3-h001b923_1
  gstreamer          conda-forge/win-64::gstreamer-1.22.3-h6b5321d_1
  icu                conda-forge/win-64::icu-72.1-h63175ca_0
  idna               conda-forge/noarch::idna-3.4-pyhd8ed1ab_0
  importlib-metadata conda-forge/noarch::importlib-metadata-6.6.0-pyha770c72_0
  importlib-resourc~ conda-forge/noarch::importlib-resources-5.12.0-pyhd8ed1ab_0
  importlib_metadata conda-forge/noarch::importlib_metadata-6.6.0-hd8ed1ab_0
  importlib_resourc~ conda-forge/noarch::importlib_resources-5.12.0-pyhd8ed1ab_0
  inflate64          conda-forge/win-64::inflate64-0.3.1-py39ha55989b_3
  intel-openmp       conda-forge/win-64::intel-openmp-2023.1.0-h57928b3_46319
  itsdangerous       conda-forge/noarch::itsdangerous-2.1.2-pyhd8ed1ab_0
  jinja2             conda-forge/noarch::jinja2-3.1.2-pyhd8ed1ab_1
  kiwisolver         conda-forge/win-64::kiwisolver-1.4.4-py39h1f6ef14_1
  krb5               conda-forge/win-64::krb5-1.20.1-heb0366b_0
  lcms2              conda-forge/win-64::lcms2-2.15-h3e3b177_1
  lerc               conda-forge/win-64::lerc-4.0.0-h63175ca_0
  libblas            conda-forge/win-64::libblas-3.9.0-17_win64_mkl
  libbrotlicommon    conda-forge/win-64::libbrotlicommon-1.0.9-hcfcfb64_8
  libbrotlidec       conda-forge/win-64::libbrotlidec-1.0.9-hcfcfb64_8
  libbrotlienc       conda-forge/win-64::libbrotlienc-1.0.9-hcfcfb64_8
  libcblas           conda-forge/win-64::libcblas-3.9.0-17_win64_mkl
  libclang           conda-forge/win-64::libclang-16.0.5-default_heb8d277_0
  libclang13         conda-forge/win-64::libclang13-16.0.5-default_hc80b9e7_0
  libdeflate         conda-forge/win-64::libdeflate-1.18-hcfcfb64_0
  libffi             conda-forge/win-64::libffi-3.4.2-h8ffe710_5
  libglib            conda-forge/win-64::libglib-2.76.3-he8f3873_0
  libhwloc           conda-forge/win-64::libhwloc-2.9.1-cpu_hadd60ae_5
  libiconv           conda-forge/win-64::libiconv-1.17-h8ffe710_0
  libjpeg-turbo      conda-forge/win-64::libjpeg-turbo-2.1.5.1-hcfcfb64_0
  liblapack          conda-forge/win-64::liblapack-3.9.0-17_win64_mkl
  libogg             conda-forge/win-64::libogg-1.3.4-h8ffe710_1
  libpng             conda-forge/win-64::libpng-1.6.39-h19919ed_0
  libsqlite          conda-forge/win-64::libsqlite-3.42.0-hcfcfb64_0
  libtiff            conda-forge/win-64::libtiff-4.5.0-h6c8260b_6
  libvorbis          conda-forge/win-64::libvorbis-1.3.7-h0e60522_0
  libwebp            conda-forge/win-64::libwebp-1.3.0-hcfcfb64_0
  libwebp-base       conda-forge/win-64::libwebp-base-1.3.0-hcfcfb64_0
  libxcb             conda-forge/win-64::libxcb-1.15-hcd874cb_0
  libxml2            conda-forge/win-64::libxml2-2.11.4-hc3477c8_0
  libxslt            conda-forge/win-64::libxslt-1.1.37-h6070c61_1
  libzlib            conda-forge/win-64::libzlib-1.2.13-hcfcfb64_4
  lxml               conda-forge/win-64::lxml-4.9.1-py39hb82d6ee_0
  m2w64-gcc-libgfor~ conda-forge/win-64::m2w64-gcc-libgfortran-5.3.0-6
  m2w64-gcc-libs     conda-forge/win-64::m2w64-gcc-libs-5.3.0-7
  m2w64-gcc-libs-co~ conda-forge/win-64::m2w64-gcc-libs-core-5.3.0-7
  m2w64-gmp          conda-forge/win-64::m2w64-gmp-6.1.0-2
  m2w64-libwinpthre~ conda-forge/win-64::m2w64-libwinpthread-git-5.0.0.4634.697f757-2
  markupsafe         conda-forge/win-64::markupsafe-2.1.3-py39ha55989b_0
  matplotlib-base    conda-forge/win-64::matplotlib-base-3.7.1-py39haf65ace_0
  mkl                conda-forge/win-64::mkl-2022.1.0-h6a75c08_874
  mrio_common_metad~ conda-forge/noarch::mrio_common_metadata-0.2.1-pyhd8ed1ab_0
  msys2-conda-epoch  conda-forge/win-64::msys2-conda-epoch-20160418-1
  multiprocess       conda-forge/win-64::multiprocess-0.70.14-py39ha55989b_3
  multivolumefile    conda-forge/noarch::multivolumefile-0.2.3-pyhd8ed1ab_0
  munkres            conda-forge/noarch::munkres-1.1.4-pyh9f0ad1d_0
  networkx           conda-forge/noarch::networkx-3.1-pyhd8ed1ab_0
  numpy              conda-forge/win-64::numpy-1.23.5-py39hbccbffa_0
  openjpeg           conda-forge/win-64::openjpeg-2.5.0-ha2aaf27_2
  openpyxl           conda-forge/win-64::openpyxl-3.1.2-py39ha55989b_0
  openssl            conda-forge/win-64::openssl-3.1.1-hcfcfb64_1
  packaging          conda-forge/noarch::packaging-23.1-pyhd8ed1ab_0
  pandas             conda-forge/win-64::pandas-2.0.2-py39h1679cfb_0
  pathos             conda-forge/noarch::pathos-0.3.0-pyhd8ed1ab_0
  patsy              conda-forge/noarch::patsy-0.5.3-pyhd8ed1ab_0
  pcre2              conda-forge/win-64::pcre2-10.40-h17e33f8_0
  peewee             conda-forge/win-64::peewee-3.16.2-py39habf9222_0
  pillow             conda-forge/win-64::pillow-9.5.0-py39ha9166d5_1
  pint               conda-forge/noarch::pint-0.21-pyhd8ed1ab_0
  pip                conda-forge/noarch::pip-23.1.2-pyhd8ed1ab_0
  platformdirs       conda-forge/noarch::platformdirs-3.5.1-pyhd8ed1ab_0
  pooch              conda-forge/noarch::pooch-1.7.0-pyha770c72_3
  pox                conda-forge/noarch::pox-0.3.2-pyhd8ed1ab_0
  ppft               conda-forge/noarch::ppft-1.7.6.6-pyhd8ed1ab_0
  psutil             conda-forge/win-64::psutil-5.9.5-py39ha55989b_0
  pthread-stubs      conda-forge/win-64::pthread-stubs-0.4-hcd874cb_1001
  pthreads-win32     conda-forge/win-64::pthreads-win32-2.9.1-hfa6e2cd_3
  py7zr              conda-forge/noarch::py7zr-0.20.5-pyhd8ed1ab_0
  pybcj              conda-forge/win-64::pybcj-1.0.1-py39ha55989b_2
  pybcpy             conda-forge/noarch::pybcpy-0.0.17-pyhd8ed1ab_0
  pycparser          conda-forge/noarch::pycparser-2.21-pyhd8ed1ab_0
  pycryptodomex      conda-forge/win-64::pycryptodomex-3.18.0-py39ha55989b_0
  pypardiso          conda-forge/noarch::pypardiso-0.4.2-pyhd8ed1ab_1
  pyparsing          conda-forge/noarch::pyparsing-3.0.9-pyhd8ed1ab_0
  pyperclip          conda-forge/noarch::pyperclip-1.8.2-pyhd8ed1ab_2
  pyppmd             conda-forge/win-64::pyppmd-1.0.0-py39h99910a6_1
  pyprind            conda-forge/win-64::pyprind-2.11.2-py39hcbf5309_1004
  pyside2            conda-forge/win-64::pyside2-5.15.8-py39hdfd5cee_3
  pysocks            conda-forge/noarch::pysocks-1.7.1-pyh0701188_6
  python             conda-forge/win-64::python-3.9.16-h4de0772_0_cpython
  python-dateutil    conda-forge/noarch::python-dateutil-2.8.2-pyhd8ed1ab_0
  python-tzdata      conda-forge/noarch::python-tzdata-2023.3-pyhd8ed1ab_0
  python_abi         conda-forge/win-64::python_abi-3.9-3_cp39
  pytz               conda-forge/noarch::pytz-2023.3-pyhd8ed1ab_0
  pyxlsb             conda-forge/noarch::pyxlsb-1.0.10-pyhd8ed1ab_0
  pyzstd             conda-forge/win-64::pyzstd-0.15.7-py39h95af829_0
  qt-main            conda-forge/win-64::qt-main-5.15.8-h2c8576c_12
  qt-webengine       conda-forge/win-64::qt-webengine-5.15.8-h5b1ea0b_0
  requests           conda-forge/noarch::requests-2.31.0-pyhd8ed1ab_0
  salib              conda-forge/noarch::salib-1.4.7-pyhd8ed1ab_0
  scipy              conda-forge/win-64::scipy-1.10.1-py39hde5eda1_3
  seaborn            conda-forge/noarch::seaborn-0.12.2-hd8ed1ab_0
  seaborn-base       conda-forge/noarch::seaborn-base-0.12.2-pyhd8ed1ab_0
  setuptools         conda-forge/noarch::setuptools-67.7.2-pyhd8ed1ab_0
  setuptools-scm     conda-forge/noarch::setuptools-scm-7.1.0-pyhd8ed1ab_0
  six                conda-forge/noarch::six-1.16.0-pyh6c4a22f_0
  soupsieve          conda-forge/noarch::soupsieve-2.3.2.post1-pyhd8ed1ab_0
  stats_arrays       conda-forge/noarch::stats_arrays-0.6.5-pyhd8ed1ab_0
  statsmodels        conda-forge/win-64::statsmodels-0.14.0-py39hbaa61f9_1
  tabulate           conda-forge/noarch::tabulate-0.9.0-pyhd8ed1ab_1
  tbb                conda-forge/win-64::tbb-2021.9.0-h91493d7_0
  texttable          conda-forge/noarch::texttable-1.6.7-pyhd8ed1ab_0
  tk                 conda-forge/win-64::tk-8.6.12-h8ffe710_0
  tomli              conda-forge/noarch::tomli-2.0.1-pyhd8ed1ab_0
  typing-extensions  conda-forge/noarch::typing-extensions-4.6.3-hd8ed1ab_0
  typing_extensions  conda-forge/noarch::typing_extensions-4.6.3-pyha770c72_0
  tzdata             conda-forge/noarch::tzdata-2023c-h71feb2d_0
  ucrt               conda-forge/win-64::ucrt-10.0.22621.0-h57928b3_0
  unicodecsv         conda-forge/noarch::unicodecsv-0.14.1-py_1
  unicodedata2       conda-forge/win-64::unicodedata2-15.0.0-py39ha55989b_0
  unidecode          conda-forge/noarch::unidecode-1.3.6-pyhd8ed1ab_0
  urllib3            conda-forge/noarch::urllib3-2.0.2-pyhd8ed1ab_0
  vc                 conda-forge/win-64::vc-14.3-hb25d44b_16
  vc14_runtime       conda-forge/win-64::vc14_runtime-14.34.31931-h5081d32_16
  voluptuous         conda-forge/noarch::voluptuous-0.13.1-pyhd8ed1ab_0
  vs2015_runtime     conda-forge/win-64::vs2015_runtime-14.34.31931-hed1258a_16
  werkzeug           conda-forge/noarch::werkzeug-2.3.4-pyhd8ed1ab_0
  wheel              conda-forge/noarch::wheel-0.40.0-pyhd8ed1ab_0
  whoosh             conda-forge/win-64::whoosh-2.7.4-py39hcbf5309_7
  win_inet_pton      conda-forge/noarch::win_inet_pton-1.1.0-pyhd8ed1ab_6
  wrapt              conda-forge/win-64::wrapt-1.15.0-py39ha55989b_0
  xlrd               conda-forge/noarch::xlrd-2.0.1-pyhd8ed1ab_3
  xlsxwriter         conda-forge/noarch::xlsxwriter-3.1.2-pyhd8ed1ab_0
  xorg-libxau        conda-forge/win-64::xorg-libxau-1.0.11-hcd874cb_0
  xorg-libxdmcp      conda-forge/win-64::xorg-libxdmcp-1.1.3-hcd874cb_0
  xz                 conda-forge/win-64::xz-5.2.6-h8d14728_0
  zipfile-deflate64  conda-forge/win-64::zipfile-deflate64-0.2.0-py39ha55989b_3
  zipp               conda-forge/noarch::zipp-3.15.0-pyhd8ed1ab_0
  zstd               conda-forge/win-64::zstd-1.5.2-h12be248_6


Proceed ([y]/n)?


Downloading and Extracting Packages

Preparing transaction: done
Verifying transaction: \
SafetyError: The package for brightway2 located at C:\Users\antoi\miniconda3\pkgs\brightway2-2.4.3-pyhd8ed1ab_0
appears to be corrupted. The path 'site-packages/brightway2/__init__.py'
has an incorrect size.
  reported size: 112 bytes
  actual size: 125 bytes


done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate ab2
#
# To deactivate an active environment, use
#
#     $ conda deactivate
```
I did another fresh install after running the following command :

`conda config --set channel_priority strict`

I obtained the same result (console output and issu with AB) but the process only took about 5 mins

**Comment by Zoophobus** [comment]
Hi, thanks for the info. I've also just found the same thing in Windows, I was wondering if you can you try and import brightway2 into the python console and confirm whether it works, or not?

**Comment by antoine7878** [comment]
When trying to import brightway2, the python console crashes after a few seconds, with apparently no error message

**Comment by Zoophobus** [comment]
thanks antoine, that's exactly what I get!

**Comment by antoine7878** [comment]
Whoa, I read about the increase perfomance of mamba but I had never tried it , I realy should have ! The installation took me about 2 min and AB is working normaly. Thank you @Zoophobus !

**Comment by AlJuel** [comment]
Thanks @Zoophobus !

Installing via Mamba also works for me - and much faster, as Antoine already mentioned!

**Comment by lucas-sys-cmd** [comment]
I met the same problem, too. however, the old installation still works. I cloned the old env to my new computer, and it works.

**Comment by jwalzberg** [comment]
I am still having that issue... I had a working version of AB in its own ab environment. I fully uninstalled annaconda and reinstalled it and still facing the same problem...

**Comment by marc-vdm** [comment]
Hi @jwalzberg I'm not sure what the issue is you mean, you say 'this issue' but also 'had a working installation'. Could you share what you were trying to do and what happened instead?

**Comment by mrvisscher** [comment]
@jwalzberg,

Could you try installing the AB using the following command:
`conda create -n ab conda-forge --solver libmamba activity-browser libxml2=2.10.4`

**Comment by jwalzberg** [comment]
I tried the above mentioned method (`conda create -n ab conda-forge --solver libmamba activity-browser libxml2=2.10.4` adding `-c` before conda-forge to avoid error) and the environment would just not solve. I tried `conda install conda-forge::activity-browser` from anaconda.org which was installing properly but still giving the same issue (AB not starting after running "activity-browser"). With the latter I tried the orginal libxml2 version and the 2.10.4 version. None worked... However, I finally thought, what about installing an earlier version (since I had one before (not sure which one)) and it was working fine. So I just chose 2.10.0 and now it's working like a charm! Not sure what was the problem exactly but I am just glad I get to use the activity-browser. Thanks @mrvisscher and @marc-vdm   for your answers! (@marc-vdm I messed up my anaconda and had to reinstall it along with all my environments).
